harness/determined/common/experimental/determined.py

```diff
@@ -380,6 +380,18 @@ def remove_oauth_client(self, client_id: str) -> None:
         except api.errors.NotFoundException:
             raise det.errors.EnterpriseOnlyError("API not found: oauth2/clients")
 
+    def stream_trials_metrics(
```

Comment:
is this meant to replace `stream_trials_training_metrics` and `stream_trials_validation_metrics`? if so, should we mark those as deprecated?

Reply:
do we have an official/preferred way of deprecating symbols in our python codebase? I recently added support for this to the cli. but that's designed for cli use. if not I can add something

Reply:
i'd just add a log warning:
```
warnings.warn(
    "stream_trials_training_metrics is deprecated, please use stream_trials_metrics instead.",
    FutureWarning,
    stacklevel=2,
)


```

Reply:
soon after my comment I added a `deprecated` decorator. I'm using that now which does log a warning with `category=DeprecationWarning`

Reply:
we're no longer logging new deprecation warnings after https://github.com/determined-ai/determined/pull/7117/commits/286efe25d30525a25012d1f2a20587ab8f971086